crates/wallet/src/wallet/mod.rs

```diff
@@ -2581,7 +2580,7 @@ macro_rules! doctest_wallet {
             .unwrap();
         let address = wallet.peek_address(KeychainKind::External, 0).address;
         let tx = Transaction {
-            version: transaction::Version::ONE,
+            version: transaction::Version::TWO,
```

Comment:
This does not create any doctest failures, though this is expected as there aren't any doctests that look for versioning, figure it is a nice thing to have to keep these tests consistent with the actual `tx_builder`